src/rules/trailingCommaRule.ts

```diff
@@ -113,7 +219,8 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
                 case ts.SyntaxKind.ConstructorType:
                 case ts.SyntaxKind.FunctionType:
                 case ts.SyntaxKind.CallSignature:
-                    this.checkListWithEndToken(node, (node as ts.SignatureDeclaration).parameters, ts.SyntaxKind.CloseParenToken);
+                    this.checkListWithEndToken(node, (node as ts.SignatureDeclaration).parameters,
+                                                ts.SyntaxKind.CloseParenToken, "functions");
```

Comment:
IIRC I have read a request to disinguish between `"arguments"` and `"parameters"` somewhere in the issues. I don't have strong opinion on this, so I leave it up to anyone else to make a decision